proxy/src/transport/tls/config.rs

```diff
@@ -291,10 +298,10 @@ pub fn watch_for_config_changes(settings: Conditional<&CommonSettings, ReasonFor
             |(mut client_store, mut server_store), ref config| {
                 client_store
                     .store(Conditional::Some(ClientConfig::from(config)))
-                    .map_err(|_| trace!("all client config watchers dropped"))?;
+                    .map_err(|_| warn!("all client config watchers dropped"))?;
```

Comment:
What's the motivation behind increasing the log level here?

Reply:
> What's the motivation behind increasing the log level here?

I will revert it.